src/test/cypress/integration/exam/ExamAssessment.spec.ts

```diff
@@ -0,0 +1,184 @@
+Cypress.on('uncaught:exception', () => {
+    return false;
+});
+
+describe('Exam Assessment', () => {
+    before('Create a course', () => {
+        cy.login(admin);
+        courseManagementRequests.createCourse().then((response) => {
+            course = response.body;
+            courseManagementRequests.addStudentToCourse(course.id, artemis.users.getStudentOne().username);
+            courseManagementRequests.addTutorToCourse(course, artemis.users.getTutor());
+        });
+    });
+
+    beforeEach('Generate new exam name', () => {
+        examTitle = 'exam' + generateUUID();
+        cy.login(admin);
+    });
+
+    afterEach('Delete Exam', () => {
+        cy.login(admin);
+        courseManagementRequests.deleteExam(exam);
+    });
+
+    after('Delete Course', () => {
+        cy.login(admin);
+        courseManagementRequests.deleteCourse(course.id);
+    });
+
+    describe('Exam exercise Assessment', () => {
+        beforeEach('Create Exam', () => {
+            prepareExam(dayjs().add(30, 'seconds'));
+        });
+
+        it('Assess a modeling exercise submission', () => {
+            courseManagementRequests.createModelingExercise({ exerciseGroup }).then((modelingResponse) => {
+                courseManagementRequests.generateMissingIndividualExams(exam);
+                courseManagementRequests.prepareExerciseStartForExam(exam);
+                cy.login(student, '/courses/' + course.id + '/exams/' + exam.id);
+                examStartEnd.startExam();
+                cy.contains(modelingResponse.body.title).should('be.visible').click();
+                modelingEditor.addComponentToModel(1);
+                modelingEditor.addComponentToModel(2);
+                modelingEditor.addComponentToModel(3);
+                cy.intercept(PUT, BASE_API + 'exercises/*/modeling-submissions').as('createModelingSubmission');
+                cy.contains('Save').click();
+                cy.wait('@createModelingSubmission');
+                cy.get('#exam-navigation-bar').find('.btn-danger').click();
+                examStartEnd.finishExam();
+                cy.login(tutor, '/course-management/' + course.id + '/exams');
+                cy.contains('Assessment Dashboard', { timeout: 60000 }).click();
+                assessmentDashboard.startAssessing();
+                modelingAssessment.addNewFeedback(2, 'Noice');
+                modelingAssessment.openAssessmentForComponent(1);
+                modelingAssessment.assessComponent(1, 'Good');
+                modelingAssessment.openAssessmentForComponent(2);
+                modelingAssessment.assessComponent(0, 'Neutral');
+                modelingAssessment.openAssessmentForComponent(3);
+                modelingAssessment.assessComponent(-1, 'Wrong');
+                assessmentDashboard.submitModelingAssessment().then((assessmentResponse) => {
+                    expect(assessmentResponse.response?.statusCode).to.equal(200);
```

Comment:
Shortwiring does not help you in this case
```suggestion
                    expect(assessmentResponse.response.statusCode).to.equal(200);
```

Reply:
I am adding this because intellij complains without it. i also dont see how it hurts the code at all.